data/graphs/FreeScale/Grammars/an_bm_cm_dn.txt

```diff
@@ -0,0 +1,11 @@
+S A S1
+S1 S D
+S A S2
+S2 X D
+X B C
+X B X1
+X1 X C
+A a
+B b
+C c
+D d
```

Comment:
It necessary to add also the original non-CNF version of grammars for non-CNF-based algorithms.
CNF increase grammar's size, what affects algorithms' performance.

Reply:
I will add this to the refactoring commit.